### DISCUSSION TITLE
Where can I download GT images from the Medical Segmentation Decathlon test datasets for performance evaluation?

### DISCUSSION THREAD

**Discussion by beargolden** [MIC-DKFZ/nnUNet#1169] [Q&A] [answered]
@FabianIsensee Thank you very much for your wonderful work and perhaps this post is an off-topic discussion on the use of nnUNet.

I have recently modified the network architecture of nnUNet and would like to evaluate the performance of the proposed network on the Task04_Hippocampus dataset, which is the smallest dataset in the Medical Segmentation Decathlon. I tried to submit our predictions on the challenge website, but failed with the following error:

>  evalutils.exceptions.FileLoaderError: Could not load and files in /input/Task01_BrainTumour with <evalutils.io.SimpleITKLoader object at 0x7fb62afc4cc0>

I guess the submission has to contain segmentation results of all the 10 tasks in a single compressed zip file, so could you tell me where I can download the ground truth images for a specific task, for example, Task04_Hippocampus?

Thanks again

**Comment by FabianIsensee** [comment, marked as answer]
This is an ongoing challenge with an online leaderboard. Nobody except the organizer has the ground truth images. If you want to know your performance on the test set you have to submit to the challenge.

I think you are right and the challenge expects all tasks to be submitted. This makes sense as this challenge is intended only (!) for algorithms that can deal with all 10 tasks. If you want to evaluate on just one dataset I would recommend you use a different challenge

**Reply by beargolden** [reply]
Many thanks~
